ganalytics.cpp

```diff
@@ -473,10 +501,12 @@ GAnalytics::GAnalytics(const QString &trackingID, QObject *parent)
 }
 
 /**
- * Destructor of class GAnalytics.
+ * Destructor of class GAnalytics. Ends the session.
  */
 GAnalytics::~GAnalytics()
 {
+    if (!trackingID().isEmpty())
+        endSession();
```

Comment:
I didn't look at the code closely enough then. I thought that all the messages in the queue are sent before the destructor finishes.

Would it make sense to do this? I thought it would be nice, as a default behavior, to just instantiate the object and already get (correct) statistics on how long the application was running on google analytics.

The alternative is of course to let the user call "startSession, endSession" and letting him wait for everything to be sent. Not very user friendly though for such "basic" and common functionality...